packages/templates/cra-template-desktop-viewer/template/src/frontend/components/modelSelector/BriefcaseStatus.scss

```diff
@@ -4,6 +4,7 @@
 *--------------------------------------------------------------------------------------------*/
 @import '@itwin/itwinui-variables/index.css';
```

Comment:
Why does this import remain but in other scss files it is removed?

Reply:
thank you, nice catch